.github/workflows/build.yml

```diff
@@ -145,10 +160,22 @@ jobs:
       - name: Setup .NET Core
         uses: actions/setup-dotnet@v3
 
+      - name: Publish Language Server
+        run: |
+          dotnet publish --configuration release ./src/Bicep.LangServer/Bicep.LangServer.csproj
+          cp -R src/Bicep.LangServer/bin/release/net7.0/publish ./src/vscode-bicep/bicepLanguageServer
+
       - name: npm ci
         run: npm ci
         working-directory: ./src/vscode-bicep
 
+      - name: Generate VSIX notice
+        run: |
+          mkdir -p ./src/vscode-bicep-notice/inputs
+          npm --prefix ./src/vscode-bicep list -a --json > ./src/vscode-bicep-notice/inputs/npm-list.json
+          cp ./src/Bicep.LangServer/obj/project.assets.json ./src/vscode-bicep-notice/inputs/project.assets.json
+          dotnet build --configuration Release ./src/vscode-bicep-notice/vscode-bicep-notice.proj
```

Comment:
I wonder if we can reuse the result of `Publish Language Server` and `Generate VSIX notice` from `build-vscode-ext` by caching them: https://docs.github.com/en/actions/using-workflows/caching-dependencies-to-speed-up-workflows#using-the-cache-action.